asm/non_matchings/overlays/actors/ovl_En_GirlA/func_80A3A8D0.s

```diff
@@ -1,202 +1,4 @@
 .rdata
-glabel D_80A3CCD0
```

Comment:
Why is some but not all of the rodata removed from this function?

Reply:
I have to do it for the data to match, I don't understand why though. Fig helped me with it.